module-infrastructure/persistence-db/src/main/kotlin/site/yourevents/guest/repository/GuestRepository.kt

```diff
@@ -34,4 +35,14 @@ class GuestRepository(
 
     override fun getReceivedInvitations(member: Member) =
         guestJPARepository.getReceivedInvitations(MemberEntity.from(member)).map(InvitationEntity::toDomain)
+
+    override fun findAttendGuestsByInvitation(invitation: Invitation): List<Guest> {
+        return guestJPARepository.findAttendGuestsByInvitation(InvitationEntity.from(invitation))
+            .map { it.toDomain() }
```

Comment:
람다식보다는 메서드 참조가 더욱 가독성을 높여줄 수 있을 것 같습니다 😄 
```suggestion
        return guestJPARepository.findAttendGuestsByInvitation(InvitationEntity.from(invitation))
            .map(GuestEntity::toDomain)
```

Reply:
아 넵 알겠습니다!! 더욱 가독성이 좋아지는 것 같습니다!!

Reply:
그렇군요!! 저는 it의 사용을 위해서 람다식을 사용했는데 메서드 참조 사용도 좋아보입니다!! 수정하였습니다.